quinn-proto/src/transport_parameters.rs

```diff
@@ -413,6 +427,23 @@ impl TransportParameters {
                     _ => return Err(Error::Malformed),
                 },
                 0xff04de1b => params.min_ack_delay = Some(r.get().unwrap()),
+                address_discovery::TRANSPORT_PARAMETER_CODE => {
+                    if !params.address_discovery_role.is_disabled() {
+                        // duplicate parameter
+                        // NOTE: this depends on the default being Disabled, which is reasonable. Is
+                        // this handled in a better way somewhere?
+                        return Err(Error::Malformed);
```

Comment:
I think it's reasonable to depend on the default being Disabled.  This is kind of specified in the spec.